client/deis.py

```diff
@@ -1043,18 +1043,35 @@ def certs_add(self, args):
         """
         Binds a certificate/key pair to an application.
 
-        Usage: deis certs:add <cert> <key>
+        Usage: deis certs:add <cert> <key> [--subject-alt-name=<san>...] [options]
```

Comment:
Is it possible to specify multiple SANs with one flag, separated by commas or something? Just a thought - some certs support up to 2000 SANs or some large number, and it may be a pain to specify them all by repeating the flag. Nitpick.